.github/workflows/publish_package.yml

```diff
@@ -1,11 +1,18 @@
-# This workflows will upload a Python Package using Twine when a release is created
+# This workflows will upload a Python Package
 # For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
 
 name: Publish Package
 
 on:
+  # publish from the Releases page:
   release:
-    types: [created]
+    types: [published]
```

Comment:
out of curiosity, concretely speaking, when will this action be triggered? should the [release workflow](https://github.com/ivadomed/ivadomed/wiki/Releases#checklist-for-creating-a-new-release) be updated if there is a change in behaviour?

Reply:
It looks like those instructions are either implying or missing "Click Publish".

Reply:
ahhhh! ok i got it now. Yes, this change totally makes sense.

Reply:
I've updated the wiki